cbits/nanovg_gl.c

```diff
@@ -1,5 +1,11 @@
 #define NANOVG_GL3_IMPLEMENTATION
 // This is used to link the implementation
+#if defined(darwin_HOST_OS)
```

Comment:
Please limit the `ifdefs` as far as possible, i.e.` don’t include the nanovg headers.